src/main/kotlin/no/nav/k9punsj/integrasjoner/k9sak/K9SakServiceImpl.kt

```diff
@@ -485,6 +488,20 @@ class K9SakServiceImpl(
         }
     }
 
+    override suspend fun hentInstitusjoner(): Pair<List<GodkjentOpplæringsinstitusjonDto>?, String?> {
```

Comment:
Fint om du unngår å bruke Pair som returtype. Prøver å gå bort ifra det da det skaper mye nullability sjekk osv.
Returener heller List<GodkjentOpplæringsinstitusjonDto> eller en tom liste ved feil :) 

Se `reserverSaksnummer` i samme klasse for liknende håndtering.